en/simulation/hitl.md

```diff
@@ -1,101 +1,133 @@
-  ./Tools/jmavsim_run.sh -q -d /dev/ttyACM0 -b 921600 -r 250
+1. Open the vehicle model's sdf file (e.g. **Tools/sitl_gazebo/models/iris/iris.sdf**).
+1. Under the `mavlink_interface plugin` section, change the `serialEnabled` and `hil_mode` parameters to `true`. 
+
+  ![HIL Parameters](../../assets/simulation/gazebo_sdf_model_hil_params.png)
+1. Replace the `serialDevice` parameter (`/dev/ttyACM0`) if necessary.
+1. Connect the flight controller to the computer and wait for it to boot.
+1. Run Gazebo in HITL mode 
+  ```sh
+  gazebo /Tools/sitl_gazebo/worlds/iris.world
   ```
+1. Start *QGroundControl*. It should autoconnect to PX4 and Gazebo.
 
-* Then run QGroundControl - it will auto-connect.
+#### jMAVSim (Quadrotor only)
 
-### Using X-Plane and QGroundControl \(Fixed Wing only\)
+> **Note** Make sure *QGroundControl* is not running!
 
-In X-Plane two key settings have to be made: In Settings -&gt; Data Input and Output, set these checkboxes:
+1. Run jMAVSim in HITL mode (replace the serial port name `/dev/ttyACM0` if necessary - e.g. on Mac OS this would be `/dev/tty.usbmodem1`):
+  ```sh
+  ./Tools/jmavsim_run.sh -q -d /dev/ttyACM0 -b 921600 -r 250
+  ```
+1. Start *QGroundControl*. It should autoconnect to PX4 and jMAVSim.
 
-![](../../assets/gcs/xplane_data_config.png)
+#### Using X-Plane (Fixed Wing only)
 
-In Settings -&gt; Net Connections in the Data tab, set localhost and port 49005 as IP address, as shown in the screenshot below:
+1. Open X-Plane
+1. In **Settings > Data Input and Output**, set these checkboxes:
 
-![](../../assets/gcs/xplane_net_config.png)
+   ![X-Plane data configuration](../../assets/gcs/xplane_data_config.png)
 
-#### Enable HITL in QGroundControl
+1. In **Settings > Net Connections**, in the *Data* tab, set localhost and port 49005 as IP address, as shown below:
 
-Widgets -&gt; HIL Config, then select X-Plane 10 in the drop-down and hit connect. Once the system is connected, battery status, GPS status and aircraft position should all become valid:
+   ![X-Plane network configuration](../../assets/gcs/xplane_net_config.png)
 
-![](../../assets/gcs/qgc_sim_run.png)
+1. Enable HITL in QGroundControl
+   1. Open *QGroundControl*
+   1. Open **Widgets > HIL Config**. Select X-Plane 10 in the drop-down and hit connect. Once the system is connected, battery status, GPS status and aircraft position should all become valid:
 
-## Fly an Autonomous Mission in HITL
+      ![](../../assets/gcs/qgc_sim_run.png)
 
-Switch to the flight planning view and put a single waypoint in front of the plane. Click on the sync icon to send the waypoint.
 
-Then select MISSION from the flight mode menu in the toolbar and click on DISARMED to arm the plane. It will take off and loiter around the takeoff waypoint.
-
-![](../../assets/gcs/qgc_sim_mission.png)
+## Fly an Autonomous Mission in HITL
 
+You should not be able to use *QGroundControl* to [run missions](../qgc/README.md#planning-missions) and otherwise control the vehicle. 
```

Comment:
FYI only, decided that we don't need crappy out of date instructions. So here we link to QGC. In future though I'd like a generic "using the simulator instructions" for stuff like flying, disabling sensors, etc. When/if that exists will link to it instead.